check.py

```diff
@@ -408,7 +408,9 @@ def fix(x):
     print '..', s
     wasm = s.replace('.s', '.wast')
     full = os.path.join('test', dot_s_dir, s)
-    actual, err = subprocess.Popen([os.path.join('bin', 's2wasm'), full], stdout=subprocess.PIPE, stderr=subprocess.PIPE).communicate()
+    start = ['--start'] if s.startswith('start_') else []
```

Comment:
in other places with similar things, we create `cmd` first, then after it we do `if condition then append to it` etc., I think that's simpler